samples/CommunityToolkit.Maui.Sample/Pages/Views/Expander/ExpanderPageCS.cs

```diff
@@ -46,7 +46,7 @@ public ExpanderPageCS()
 
 						new Button()
 							.Text("Learn more")
-							.Invoke(button => button.Clicked += async (s, e) => await Launcher.OpenAsync("https://dot.net/maui"))
+							.Invoke(button => button.Clicked += async (s, e) => await Launcher.OpenAsync($"https://dot.net/maui"))
```

Comment:
The `$` symbol isn't needed here. In fact it isn't needed in any of the changes in this PR

Reply:
It is purely to remove warning message. I get ADHD about yellow warning on every line like that.

Reply:
What warning is that?

Reply:
S1075 Refactor your code not to use absolute URL's. Adding the above remove the error message.

Reply:
I think one of these 3 options are a better fix for that: https://stackoverflow.com/a/65465628/32348

Mainly because someone will likely just remove the `$` because nothing is being interpolated into the string and then reintroduce the warning.

Reply:
Could not use inline variable. Same error.

Reply:
Went with suppresion